Engine/source/windowManager/platformWindowMgr.h

```diff
@@ -136,6 +136,10 @@ class PlatformWindowManager
    /// This method indicates to created windows to show as normal.
    virtual void setDisplayWindow(bool set){}
 
+   /// Checks if DWM is running (Windows only)
+   virtual bool isDesktopCompositionEnabled() { return false; }
+   virtual void updateDesktopCompositionState() {}
```

Comment:
The alternative is to dynamic_cast the window manager before using it in gfxDevice, but that seems worse to me - I would prefer not to add more reliance on RTTI. My thinking was that it is Windows-only for now, but there might be similar mechanisms on other platforms that people might want to slot in.

In general, this is the old problem of using inheritance hierarchies to model static polymorphism.